package.json

```diff
@@ -7,6 +7,7 @@
     "access": "public"
   },
   "main": "index.js",
+  "type": "module",
```

Comment:
Should we take this opportunity to go to `1.0.0` like we recently did with `eslint-config-brightspace`?

Reply:
I think that would be worthwhile.